src/skycoin/config.go

```diff
@@ -0,0 +1,278 @@
+package skycoin
+
+import (
+	"flag"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"log"
+
+	"github.com/skycoin/skycoin/src/cipher"
+	"github.com/skycoin/skycoin/src/util/file"
+)
+
+// Config records fiber coin configs
+type Config struct {
+	Blockchain BlockchainConfig
+	Node       NodeConfig
+	Build      BuildConfig
+}
+
+// BuildConfig records build info
+type BuildConfig struct {
+	Version string // version number
+	Commit  string // git commit id
+	Branch  string // git branch name
+}
+
+// BlockchainConfig records genesis data
+type BlockchainConfig struct {
```

Comment:
All of these can be merged into a single Config struct.  If we don't want to expose any as CLI parameters, we just won't register them with `flag`